Assets/UI/Popups/greatpeoplepopup.lua

```diff
@@ -505,21 +538,27 @@ function ViewCurrent( data:table )
         scrollAmt = math.clamp( scrollAmt, 0, 1);
         Controls.PeopleScroller:SetScrollValue( scrollAmt );
     end
+    if IsTutorialRunning() then
```

Comment:
this matches the code found in the unmodified Firaxis version of this file, has no adverse impact on the CQUI changes